pontoon/base/static/css/translate.css

```diff
@@ -696,7 +707,7 @@ body > header aside p {
   color: #AAAAAA;
   display: none;
   font-size: 16px;
-  left: 43px;
+  left: 53px;
```

Comment:
@mathjazz It looks better, but when there is caret, it looks really bad. But i can hide caret when menu is opened, what do you think?